recipes/_freebsd.rb

```diff
@@ -21,4 +21,5 @@
   package 'devel/gmake'
   package 'devel/autoconf'
   package 'devel/m4'
+  package 'lang/gcc49' if node['os_version'].to_i <= 903_000
```

Comment:
If that seems easier to follow than using this large `os_version` number. Although I just remembered FreeBSD has crazy platform version numbers like `9.3-RELEASE-p13`. I've got nothing.